### PR TITLE
Topic map edits for ARO content

### PR DESCRIPTION
**Pull request by dhoyle-rh**
I know we use the _distro_map.yml file in master, but I added a section for ARO here so I could test the output.

### CODE REVIEW

**Comment by dhoyle-rh** [comment]
@adellape, this is ready for review

File: _distro_map.yml [hunk 1]
```diff
@@ -76,3 +76,13 @@ atomic-registry:
     master:
       name: 'Latest'
       dir: latest
+openshift-arho:
```
Comment (adellape) [inline _distro_map.yml]: I've switched it to `openshift-aro` everywhere in https://github.com/openshift/openshift-docs/pull/14652, if you can update throughout in this PR, too.

**Comment by dhoyle-rh** [comment]
It be done...

**Review by adellape — APPROVED** [review]
LGTM :+1:

**Comment by adellape** [comment]
cc @vikram-redhat

**Comment by vikram-redhat** [comment]
@adellape need to take out _distro_map.yml from this.

**Comment by adellape** [comment]
Closing in favor of https://github.com/openshift/openshift-docs/pull/14706, which carries these changes.

**Pull request closed without merging**